packages/runtime/src/helpers/utils.ts

```diff
@@ -268,6 +267,17 @@ export const findModuleFromBase = ({ paths, candidates }): string | null => {
       // Ignore the error
     }
   }
+  // if we couldn't find a module from paths, let's try to resolve from here
+  for (const candidate of candidates) {
+    try {
+      const modulePath = require.resolve(candidate)
+      if (modulePath) {
+        return modulePath
+      }
+    } catch {
+      // Ignore the error
+    }
+  }
```

Comment:
[sand] instead of an additional for loop, consider pushing the current directory `'.'` to the `paths` array

Reply:
I tried, and it's actually not that simple. Using just `.`, `__dirname` etc in `paths` alone doesn't replicate behavior of `require.resolve` (when using default `paths`). We can get what default would use by calling `require.resolve.paths(candidate)` and we can push all that to our custom `paths` sure, but I never actually use that function before in production and I'm not sure of its sharp edges so would prefer to avoid going that route.

Just for some more details on that - here's example what default paths look like generally - it first starts with directory of current module + `/node_modules` and then go up directory tree, later seems to be checking some globals (??) etc:
```
> pwd
/Users/misiek/dev/next-runtime
> node -e "console.log(require.resolve.paths('next'))"
[
  '/Users/misiek/dev/next-runtime/node_modules',
  '/Users/misiek/dev/node_modules',
  '/Users/misiek/node_modules',
  '/Users/node_modules',
  '/node_modules',
  '/Users/misiek/.node_modules',
  '/Users/misiek/.node_libraries',
  '/Users/misiek/.nvm/versions/node/v18.14.0/lib/node'
]
```

So TL/DR it's not as simple as `.` and it might potentially introduce unknown problems that I would prefer to avoid

Reply:
Thanks for the explanation @pieh, got it 👍🏼